src/Observers/Loggers/CategoryObserver.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Bonnier\WP\Redirect\Observers\Loggers;
+
+use Bonnier\WP\Redirect\Helpers\UrlHelper;
+use Bonnier\WP\Redirect\Models\Log;
+use Bonnier\WP\Redirect\Observers\AbstractObserver;
+use Bonnier\WP\Redirect\Observers\CategorySubject;
+use Bonnier\WP\Redirect\Observers\Interfaces\SubjectInterface;
+
+class CategoryObserver extends AbstractObserver
+{
+    /**
+     * @param SubjectInterface|CategorySubject $subject
+     * @throws \Exception
+     */
+    public function update(SubjectInterface $subject)
+    {
+        if ($subject->getType() === CategorySubject::UPDATE && $category = $subject->getCategory()) {
+            $log = new Log();
+            $log->setSlug(UrlHelper::normalizePath(get_category_link($category->term_id)))
```

Comment:
You can remove the call to `UrlHelper::normalizePath()` if you move it inside the `setSlug()`
```suggestion
            $log->setSlug(get_category_link($category->term_id))
```